plugins/javadoc/src/main/kotlin/javadoc/renderer/KorteJavadocRenderer.kt

```diff
@@ -0,0 +1,186 @@
+package javadoc.renderer
+
+import com.soywiz.korte.*
+import javadoc.JavadocLocationProvider
+import javadoc.pages.*
+import javadoc.renderer.JavadocContentToHtmlTranslator.Companion.buildLink
+import kotlinx.coroutines.CoroutineScope
+import kotlinx.coroutines.Dispatchers
+import kotlinx.coroutines.launch
+import kotlinx.coroutines.runBlocking
+import org.jetbrains.dokka.base.renderers.OutputWriter
+import org.jetbrains.dokka.links.DRI
+import org.jetbrains.dokka.pages.*
+import org.jetbrains.dokka.plugability.DokkaContext
+import org.jetbrains.dokka.renderers.Renderer
+import org.jetbrains.kotlin.utils.addToStdlib.safeAs
+import java.nio.file.Path
+import java.nio.file.Paths
+import java.time.LocalDate
+
+typealias TemplateMap = Map<String, Any?>
+
+class KorteJavadocRenderer(val outputWriter: OutputWriter, val context: DokkaContext, val resourceDir: String) :
+    Renderer {
+    private lateinit var locationProvider: JavadocLocationProvider
```

Comment:
NIT: I _personally_ mislike this style. I certainly can see the benefits (you do not have to pass this around everywhere), but I think a similar concept like done with the `InnerTranslator` in `JavadoccontentToTemplateMapTranslator` is better, since it is extremly simple to reason about and the new "context" provided by this other class forbids a lot of dangerous things.

Reply:
Actually this is just to be consistent with DefaultRenderer

Reply:
I think its still worth it to go the other style ("InnerTranslator" like), but this is fully your choice in the end 👍